JSON/README.md

```diff
@@ -13,45 +13,42 @@ JSON schemas for all the config files and objects used by Tyk products.
 
 ## Client to Gateway Authentication
 
-| Type        | JSON      | YAML | Comments | Draft |
-| ----------- | --------- | ---- | --------- | -------- |
-| Tyk API definition | ⚠️ | ❌️ | - | 07 |
-| Tyk API OAS definition | ❌️ | ❌️ | - | 07 |
-| Tyk API definition in Tyk operator | ❌️ | ❌ | - | 07 |
-| Tyk key definition | ⚠️ | ❌️ | Also referred as "session object" | 07 |
-| Tyk policy definition | ❌️ | ❌️ | - | 07 |
-| Tyk OSS gateway config file | ⚠️ | NA | - | 07 |
-| Tyk Pro gateway config file | ❌️ | NA | - | 07 |
-| Tyk hybrid gateway config file | ❌️ | NA | - | 07 |
-| Tyk pump config file | ❌️ | NA | - | 07 |
-| Tyk dashboard config file | ❌️ | NA | - | 07 |
-| Tyk Identity broker config file | ❌️ | NA | - | 07 |
+| Schema Description        | JSON      | YAML | Comments | Draft | When is it used
+| ----------- | --------- | ---- | --------- | -------- | -------- |
+| Tyk Gateway API definition | ⚠️ | ❌️ | - | 07 | An OpenAPI Spec object |
```

Comment:
why gateway ApiDefinition is marked as `untested` in JSON? I guess we can mark it as ✅, unless I miss something since it is already used by vs code extension.